demo/chatbot_simple_demo/run.py

```diff
@@ -1,17 +1,25 @@
 import gradio as gr
 import random
-
-def respond(chat_history, message):
-  response = random.choice(["Yes", "No"])
-  return chat_history + [[message, response]]
+import time
 
 with gr.Blocks() as demo:
     chatbot = gr.Chatbot()
     msg = gr.Textbox()
     clear = gr.Button("Clear")
 
-    msg.submit(respond, [chatbot, msg], chatbot)
+    def user_message(message, history):
+        return history + [[message, None]]
+
+    def bot_message(history):
+        response = random.choice(["Yes", "No"])
+        history[-1][1] = response
+        time.sleep(1)
+        return history
+
+    msg.submit(user_message, [msg, chatbot], chatbot).then(
```

Comment:
Love it! Let's update the other chatbot demos to use this syntax as well?

Reply:
Done.

Reply:
updated demo/chatbot_multimodal, demo/chatbot was too complicated